pkg/gpio/gpio_linux.go

```diff
@@ -39,10 +40,49 @@ func (v Value) String() string {
 	return "1"
 }
 
+// GetPinID is used to get the ID for a specific GPIO. This is done by
```

Comment:
the first sentence should be an active voice summary of what this does, e.g.

```go
// GetPinID computes the sysfs pin number for a specific port on a specific GPIO chip.
```

Reply:
Done